src/embed/sphere-renderer.js

```diff
@@ -148,7 +157,30 @@ SphereRenderer.prototype.createPhotosphere_ = function(texture, opt_params) {
     }
   }
 
-  var material = new THREE.MeshBasicMaterial({ map: texture });
+  var material;
+  if (texture.format === THREE.RGBAFormat && texture.flipY === false) {
+    material = new THREE.ShaderMaterial({
```

Comment:
iOS has a rendering issue for HLS stream. It can be fixed when we use RGBAFormat. But RGBAFormat rendering has a color issue. So we also need a custom material to correct it.